src/pages/tezos/index.js

```diff
@@ -76,6 +78,10 @@ const TestimonialCard = ({ link, img, name, username, text }) => {
 };
 
 function Landing() {
+  useEffect(() => {
+    trackEvent('Homepage-View');
```

Comment:
No need to specifically track a particular view. Amplitude does that by default.